cocotb/config.py

```diff
@@ -45,11 +46,16 @@
 import cocotb
 import cocotb._vendor.find_libpython as find_libpython
 
-__all__ = ["share_dir", "makefiles_dir"]
+__all__ = ["share_dir", "makefiles_dir", "libs_dir"]
 
 
 share_dir = os.path.join(os.path.dirname(cocotb.__file__), "share")
 makefiles_dir = os.path.join(os.path.dirname(cocotb.__file__), "share", "makefiles")
+libs_dir = os.path.join(os.path.dirname(cocotb.__file__), "libs")
+
+# On Windows use mixed mode "c:/a/b/c" as this work in all cases
+if os.name == "nt":
+    libs_dir = libs_dir.replace("\\", "/")
```

Comment:
Doesn't this only work in powershell? It might be worth noting (somewhere) that only bash and powershell are currently supported. I don't think that it's currently listed anywhere.

Reply:
Both `\` and `/` are supposed on widows for very long time.

Reply:
Looks like Windows supports forward slashes at the API level, but cmd.exe doesn't; so it depends if the path goes in as a value, or if cmd.exe parses it first. I guess we shouldn't worry about this for now.
https://stackoverflow.com/questions/10523708/why-does-the-cmd-exe-shell-on-windows-fail-with-paths-using-a-forward-slash